backend/src/packages/users/users.ts

```diff
@@ -4,9 +4,12 @@ import { UserController } from './user.controller.js';
 import { UserModel } from './user.model.js';
 import { UserRepository } from './user.repository.js';
 import { UserService } from './user.service.js';
+import { UserDetailsModel } from './user-details.model.js';
+import { UserDetailsRepository } from './user-details.repository.js';
 
 const userRepository = new UserRepository(UserModel);
-const userService = new UserService(userRepository);
+const userDetailsRepository = new UserDetailsRepository(UserDetailsModel);
+const userService = new UserService({ userRepository, userDetailsRepository });
```

Comment:
After removing userDetailsRepository as @andanf-e suggested, don't forget to made UserService constructor get not object, but single argument - userRepository.